DashAI/front/src/App.test.jsx

```diff
@@ -2,15 +2,17 @@ import React from 'react';
 // import { render, screen } from '@testing-library/react';
 import { render } from '@testing-library/react';
 import { BrowserRouter as Router } from 'react-router-dom';
-import { ThemeProvider } from 'styled-components';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
 import '@testing-library/jest-dom';
 import App from './App';
 import theme from './styles/theme';
 
+const myTheme = createTheme(theme);
+
 test('loads and displays model', async () => {
   render(
     <Router>
-      <ThemeProvider theme={theme}>
+      <ThemeProvider theme={myTheme}>
```

Comment:
It seems to me that besides wrapping the app with the theme, it is also a good idea to add X, since it standardizes the default css according to MUI styles.

```suggestion
      <ThemeProvider theme={myTheme}>
          <CssBaseline />
```

Source: 
- https://github.com/mui/material-ui/blob/master/examples/material-cra/src/index.js
- https://mui.com/material-ui/react-css-baseline/

Reply:
Thank you!
I added the CssBaseline component along with a new MUI global style because some CSS rules were being overwritten.